src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -242,8 +248,12 @@ export class CalciteTipManager {
       return <Host />;
     }
     return (
-      <Host onKeydown={this.tipManagerKeyDownHandler}>
-        <div tabindex="0">
+      <Host>
+        <div
+          tabindex="0"
+          onKeyUp={this.tipManagerKeyDownHandler}
+          ref={(el) => (this.container = el as HTMLDivElement)}
```

Comment:
Can you create a private method for this to avoid creating multiple functions on each render?